project/Dependencies.scala

```diff
@@ -1,17 +1,17 @@
 import sbt._
 
 object Dependencies {
+  val scalacheckVersion = "1.14.3"
+  val akkaHttpVersion = "10.2.0"
+  val akkaStreamsVersion = "2.6.9"
```

Comment:
I'm slightly hesitant to upgrade the Akka dependencies without testing them against the product in which we use this library, let's separate the concerns of doing cross compile and upgrading dependencies in 2 PRs

Reply:
Generally speaking, Akka libs are compatible from 2.4 to 2.6, but I can separate that.